contracts/deploy/021_solidity_upgrade.js

```diff
@@ -0,0 +1,322 @@
+    const cCompoundStrategyProxy = await ethers.getContractAt(
+      "CompoundStrategyProxy",
+      dCompoundStrategyProxy.address
+    );
+    const dCompoundStrategy = await deployWithConfirmation("CompoundStrategy");
+    log("Deployed CompoundStrategy...");
+    await withConfirmation(
+      cCompoundStrategyProxy.connect(sDeployer)[
+        // eslint-disable-next-line
+        "initialize(address,address,bytes)"
+      ](dCompoundStrategy.address, deployerAddr, [])
+    );
+    log("Initialized CompoundStrategyProxy...");
+    const cCompoundStrategy = await ethers.getContractAt(
+      "CompoundStrategy",
+      dCompoundStrategyProxy.address
+    );
+    await withConfirmation(
+      cCompoundStrategy
+        .connect(sDeployer)
+        ["initialize(address,address,address,address[],address[])"](
+          addresses.dead,
+          cVaultProxy.address,
+          assetAddresses.COMP,
+          [assetAddresses.USDC, assetAddresses.USDT],
+          [assetAddresses.cUSDC, assetAddresses.cUSDT]
+        )
+    );
+    log("Initialized CompoundStrategy...");
+    // Transfer CompoundStrategy governance to governor
+    await withConfirmation(
+      cCompoundStrategy.connect(sDeployer).transferGovernance(cGovernor.address)
+    );
+    log("Transferred governance of CompoundStrategy...");
+
+    /**
+     *
+     * AAVE strategy
+     *
+     */
+    const dAaveStrategyProxy = await deployWithConfirmation(
+      "AaveStrategyProxy",
+      [],
+      "InitializeGovernedUpgradeabilityProxy"
+    );
+    log("Deployed AaveStrategyProxy...");
+    const cAaveStrategyProxy = await ethers.getContract("AaveStrategyProxy");
+    const dAaveStrategy = await deployWithConfirmation("AaveStrategy");
+    log("Deployed AaveStrategy");
+    const cAaveStrategy = await ethers.getContractAt(
+      "AaveStrategy",
+      dAaveStrategyProxy.address
+    );
+    await withConfirmation(
+      cAaveStrategyProxy
+        .connect(sDeployer)
+        ["initialize(address,address,bytes)"](
+          dAaveStrategy.address,
+          deployerAddr,
+          []
+        )
+    );
+    log("Initialized AaveStrategyProxy...");
+    const initFunction =
+      "initialize(address,address,address,address[],address[],address,address)";
+    await withConfirmation(
+      cAaveStrategy
+        .connect(sDeployer)
+        // eslint-disable-next-line
+        [initFunction](
+          assetAddresses.AAVE_ADDRESS_PROVIDER,
+          cVault.address,
+          assetAddresses.AAVE,
+          [assetAddresses.DAI],
+          [assetAddresses.aDAI],
+          assetAddresses.AAVE_INCENTIVES_CONTROLLER,
+          assetAddresses.STKAAVE
+        )
+    );
+    log("Initialized AaveStrategy...");
+
+    await withConfirmation(
+      cAaveStrategy.connect(sDeployer).transferGovernance(cGovernor.address)
+    );
+    log("Transferred governance of AaveStrategy...");
+
+    // Flipper
+    await deployWithConfirmation(
+      "Flipper",
```

Comment:
Did the flipper change?

Reply:
Yes it did with the Solidity upgrade and use of `immutable`. We don't have to redeploy it though.

Reply:
Yeah, I think we can just keep the live version in use.

Reply:
Removed ✅